src/index.css

```diff
@@ -8,3 +8,21 @@
 * {
   box-sizing: border-box;
 }
+
+.contentEl{
+  text-align: center;
+}
+
+
+
+.answerStatusEl{
```

Comment:
Please use `#exam2 .answerStatusEl`, so this style doesn't apply to any other pages.